src/js/views/dashboard/widget/wizard/Steps/Filters/index.jsx

```diff
@@ -126,7 +126,7 @@ const Index = props => {
 
   const handleChangeDynamicValue = useCallback(event => {
     const { value } = event.target;
-    setData(state => ({ ...state, dynamicValue: value }));
+    setData(state => ({ ...state, dynamicValue: value.replace(/\D/gim, '') }));
```

Comment:
![image](https://user-images.githubusercontent.com/32846891/98395524-27556800-203b-11eb-9015-77cba67e5e77.png)

Also add negative number validation to the input.
This prevents the alternating behavior between 0 and 1 from occurring.

Reply:
Done

Reply:
Please also place the validation in the field id = 'lastRegs'